### DISCUSSION TITLE
How to prevent the scan popup from disappearing too soon?

### DISCUSSION THREAD

**Discussion by Sphekes** [xiaoyifang/goldendict-ng#1539] [Q&A]
When using the scan popup, if your mouse leaves the popup window, the popup disappears immediately, this did not exist in old goldendict and is very inconvinient. How to set a time keeping the popup, or cancel the disappearing once for all. Thanks in advance.

**Comment by shenlebantongying** [comment]
> disappearing once for all

Click the red pin.

![image](https://github.com/xiaoyifang/goldendict-ng/assets/20123683/782d7f33-9bef-407d-97f1-e830fcbaf08e)

**Reply by Sphekes** [reply]
Thanks for your answer. And I forgot to say, I'm using Arch Linux with KDE.

In my DE, clicking the red pin will turn the popup to main window, which won't disappear automatically after the lookup, this is not what I'm expecting. So I wonder if "not disappearing automatically if mouse leaves" could be the default behavoir just like old gd, because the popup windows will go anyway after losing its focus, so there's no point to make it disappear immediately after mouse leaves.
